internal/pkg/manifest/records.go

```diff
@@ -55,11 +54,6 @@ func (r *Records) SetRecords(records []model.ObjectManifest) error {
 		return errors
 	}
 
-	// Validate
-	if err := r.validate(); err != nil {
-		return err
-	}
-
```

Comment:
`manifest.Records` je spolocny kod pre project/template manifest.
Je to zoznam vsetkych zaznamov, branch/config/rows manifestov.

Bola tu metoda `validate`, ktora je ale duplicitna.

Project ma svoju validaciu:
- pri nacitani https://github.com/keboola/keboola-as-code/blob/c7679c76e4fd5056240858e51f0eba5a86bf1966/internal/pkg/project/manifest/content.go#L56-L59
- aj pri ulozeni: https://github.com/keboola/keboola-as-code/blob/c7679c76e4fd5056240858e51f0eba5a86bf1966/internal/pkg/project/manifest/content.go#L65-L68

Template ma tiez svoju validaciu:
- pri nacitani https://github.com/keboola/keboola-as-code/blob/c7679c76e4fd5056240858e51f0eba5a86bf1966/internal/pkg/template/manifest/file.go#L100-L103
- pri ulozeni https://github.com/keboola/keboola-as-code/blob/c7679c76e4fd5056240858e51f0eba5a86bf1966/internal/pkg/template/manifest/file.go#L110-L112

-----------------------
V project/tempalte sa validuje rozdielne `required_in_project`, a museli by sme to pridat aj do spolocneho `manifest.Records`, co by bolo ale zbytocen duplicitne.